test/library/packages/UnitTest/PopulateTests_PRIM/TestCountInt.chpl

```diff
@@ -1,3 +1,7 @@
+/*
+  verifies that only those functions are counted which
+  takes an `int` type as an argument.
```

Comment:
This may be just a "Lydia is a verbose person" thing, but I would say a little more in this comment about why we're expecting the int version to be important (e.g. "because an int was passed to the primitive")

Reply:
I think you are correct. This will help us later.